ecpy_pulses/sequences/views/sequence_editor_view.enaml

```diff
@@ -0,0 +1,492 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright 2015-2016 by EcpyPulses Authors, see AUTHORS for more details.
+#
+# Distributed under the terms of the BSD license.
+#
+# The full license is in the file LICENCE, distributed with this software.
+# -----------------------------------------------------------------------------
+"""
+"""
+from __future__ import (division, unicode_literals, print_function,
+                        absolute_import)
+
+from atom.api import (Atom, Typed, Unicode, Constant, Bool, Value, Dict,
+                      Subclass, set_default)
+from enaml.core.declarative import d_, d_func
+from enaml.layout.api import vbox, hbox, align, spacer
+from enaml.widgets.api import (PushButton, Container, Label, Field,
+                               GroupBox, ScrollArea, Menu, Action, Stack,
+                               StackItem, Form, Notebook, Page, CheckBox,
+                               ObjectCombo, Splitter, SplitItem, HGroup)
+from enaml.core.api import Looper, Conditional, Include
+from enaml.workbench.core.core_plugin import CorePlugin
+
+from ecpy.utils.widgets.dict_editor import (DictEditor, FieldFieldEditor,
+                                            FieldFieldCompleterEditor)
+from ecpy.utils.widgets.qt_completers import QtLineCompleter
+from ...utils.entry_eval import EVALUATER_TOOLTIP
+# from ecpy.utils.widgets.list_editor import PopupListMenu
+from ecpy.utils.container_change import ContainerChange
+from ...pulse import Pulse
+
+from itertools import chain
+from functools import partial
+
+from ..base_sequences import AbstractSequence
+from ...infos import SequenceInfos, PulseInfos
+
+#: TODO A Nice thing wold be to make the stock ecpy popuplistmenu more flexible.
```

Comment:
Do we need the distinction add pulse add sequence in the visiblity ? What would it take to make the standard menu work here ?